pkg/workloadmeta/collectors/internal/kubeapiserver/pod_test.go

```diff
@@ -137,5 +137,5 @@ func Test_PodsFakeKubernetesClient(t *testing.T) {
 			},
 		},
 	}
-	testFakeHelper(t, createResource, newPodStore, expected)
+	testCollectEvent(t, createResource, newPodStore, expected)
```

Comment:
I couldn’t find where this `testCollectEvent` function is defined:

```
lenaic@lenaic-Precision-5570:~/doc/devel/DataDog/datadog-agent (ali/fix-flaky-deployment-test $% u= origin/ali/fix-flaky-deployment-test)$ git grep testCollectEvent

pkg/workloadmeta/collectors/internal/kubeapiserver/pod_test.go:140:     testCollectEvent(t, createResource, newPodStore, expected)
```

Reply:
Thanks for catching that, I renamed `testFakeHelper` but had a bug with VSCode. The file wasn't saved and couldn't be closed. It should be fixed now